demo/DemoClient.js

```diff
@@ -1,27 +1,40 @@
+/*
+ * For the _startPopupUrlParams to work ../src/main.ts must:
+ * export * from './UrlRpcEncoder';
+ * export { ResponseMethod } from './Messages';
+ */
 class DemoClient {
     constructor() {
         this._connected = this._startIFrame();
-        this._redirectClient = new Rpc.RedirectRpcClient('second.html', DemoClient.DEMO_ORIGIN);
+        this._redirectClient = new Rpc.RedirectRpcClient('http://rpc.local/demo/second.html', DemoClient.DEMO_ORIGIN);
```

Comment:
Would be good to add some quick instructions or a comment on how to actually run this demo.